index.bs

```diff
@@ -4721,9 +4721,9 @@ Registry Definitions</h4>
 <h4 id=reg-pub>
 Publishing Registries</h4>
 
-	[=Registries=] can be published either
+	[=Registries=] can be published by [=chartered group|chartered=] or [=elected groups=] either
 	as a stand-alone [=technical report=] on the [=Registry Track=] called a <dfn>registry report</dfn>,
-	or incorporated as part of a [=Recommendation=] as an <dfn oldids="registry-section">embedded registry</dfn>.
+	or, in the case of [=Working Groups=], incorporated as part of a [=Recommendation=] as an <dfn oldids="registry-section">embedded registry</dfn>.
```

Comment:
```suggestion
	or, in the case of those owned by [=Working Groups=],
	incorporated as part of a [=Recommendation=] as an <dfn oldids="registry-section">embedded registry</dfn>.
```

Reply:
Indirectly, I think which phrasing we pick here kind of depends on what we do on line 4727: depending on how we right the two parts of this contrast, we might suggest that doing a registry in a REC is the only way for WGs to do it, while we actually want to say that only WGs have the ability to do so, but that they can also do it standalone like everyone else.